docs/OMS-Agent-for-Linux.md

```diff
@@ -686,7 +690,7 @@ The default settings are
 ```
 
 # Uninstalling the OMS Agent for Linux
-The agent packages can be uninstalled using dpkg or rpm, or by running the bundle .sh file with the `--remove` argument. Additionally, if you want to completely remove all pieces of the OMS Agent for Linux you can run the bundle .sh file with the `--purge` arguement. 
+The agent packages can be uninstalled using dpkg or rpm, or by running the bundle .sh file with the `--remove` argument. Additionally, if you want to completely remove all pieces of the OMS Agent for Linux you can run the bundle .sh file (or onboard_agent.sh file) with the `--purge` argument.
```

Comment:
hi, can you change the "remove all pieces" to be "to remove user data such as certificate and agent configurations"?

Reply:
Done